sdk/servicebus/Microsoft.Azure.ServiceBus/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
-# 4.1.1
+## 4.1.1
```

Comment:
This doesn't look right to me.   Did you intend to give the version a double `##`?

We've got a parent-child relationship between the `version` and `change category` under it, but the double-pound on the version makes them siblings.   If the change was intended, the pattern wasn't applied to earlier versions in the file, only the two most recent.

Reply:
Going forward we want to follow [changelog guidiance](https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#changelog-guidance) so we only use the double # for specifying the versions.
I changed the other siblings to be bold tags rather than headers.

Reply:
Fair enough, though unless I overlooked it, it seems the guidelines call out using `#` for versions and only specify not adding another L1 header, which I would associate with `#` or an `<h1>` tag.   

![image](https://user-images.githubusercontent.com/913445/70261248-0831c480-1760-11ea-8b38-c29576811190.png)

Wouldn't that indicate that the existing format is valid in this case?

Reply:
Right, will update.